app/views/admin/pages/home.html.erb

```diff
@@ -16,12 +16,12 @@
       <div><%= "You\'re an admin, #{current_user.first_name}" if current_user&.admin? %></div>
       <div>
         <h4>All Users</h4>
-        <% for user in @users %>
-          <div>
-            <p><%= "#{ user.first_name } #{ user.last_name } (#{ user.role })" %></p>
-            <p><%= user.email %></p>
-            <hr>
-          </div>
-        <% end %>
+        <% @users.each { |user| %>
```

Comment:
please do not use `{}` for multiline blocks

Reply:
Alright